.drone.yml

```diff
@@ -0,0 +1,51 @@
+pipeline:
+   node 9:
+      image: node:9.10.1
+      secrets:
+         - password
+      commands:
+         - export BITGOJS_TEST_PASSWORD=$PASSWORD
```

Comment:
This doesn't need to be defined unless we are running the integration tests.

Also, if the secret were called `BITGOJS_TEST_PASSWORD`, we would not need to specially set this. I'd also prefer that secrets are stored in the .drone.yml file, so that a PR is required to change them.

Reply:
> This doesn't need to be defined unless we are running the integration tests.

Oops, sorry, this part is incorrect. We do need it defined all the time, so it's fine to leave it here.